doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -388,18 +384,17 @@ damage through unauthorized access. It is however still a good idea
 documentation <https://rforge.net/Rserve/doc.html>`_ for more
 information on password encryption and access security.
 
-You should already have the following 4 JVM options added to your
-:fixedwidthplain:`domain.xml` by the Dataverse installer::
+You should have the following 4 file options::
 
-        <jvm-options>-Ddataverse.rserve.host=localhost</jvm-options>
-        <jvm-options>-Ddataverse.rserve.port=6311</jvm-options>
-        <jvm-options>-Ddataverse.rserve.user=rserve</jvm-options>
-        <jvm-options>-Ddataverse.rserve.password=rserve</jvm-options>
+        RserveHost=localhost
```

Comment:
I would add `RserveConfigured=true` because without that some things will not be rendered on ui